src/data/contributors/harikumar.json

```diff
@@ -0,0 +1,10 @@
+{
+    "name": "Hari kumar",
+    "skills" : ["JS", "c++"],
+    "cause" : "I code for knowledge",
```

Comment:
I code for is already the part of pre-written string, you have to just write the cause :)